php/create-team.php

```diff
@@ -0,0 +1,33 @@
+<?php
+	require 'connect-db.inc';
+
+	$db = connect_db();
+	$name = $_POST ['name'];
+	$manager_id = $_SESSION ['user_id'];
+
+	$stmt = $db->prepare('SELECT COUNT(*) FROM team WHERE name = ? AND manager_id = ?');
+	$stmt->bind_param('si', $name, $manager_id);
+	$stmt->execute();
+	$stmt->bind_result($result);
+
+    if ($result == 0)
```

Comment:
You miss $stmt->fetch() to get the result of the statement. Also, don't compare against `0`, but `'0'`